src/test/java/com/bitmovin/api/examples/util/H264Representation.java

```diff
@@ -11,6 +11,9 @@
  */
 public class H264Representation
 {
+    private String outputBasePath;
```

Comment:
Why do we need that? That should probably not be here.
The H264Representation does not care about inputs nor outputs.

Reply:
That is a Helper class that is needed to keep track of the generated representations for the manifest generations.